website/docs/migration/migrating-from-mongodb.md

```diff
@@ -2,9 +2,9 @@
 sidebar_position: 2
 ---
 
-# Migrating from MongoDB
+# Migration
 
-Before you begin this section of the migration process, go through the pre-migration process so as to ensure a successful migration from MongoDB to FerretDB:
+Before you begin this section of the migration process, go through the pre-migration process so as to ensure a successful migration:
```

Comment:
How can one _begin_ the section? Only if "section" is some event that should be started. I guess you missed a word like "reading" or "following" or something.